src/reporting/ReputationToken.sol

```diff
@@ -11,19 +11,17 @@ import 'ROOT/reporting/Interfaces.sol';
 contract ReputationToken is DelegationTarget, Typed, Initializable, StandardToken, IReputationToken {
     using SafeMath for uint256;
 
-    string public name;
-    string public symbol;
-    uint256 public decimals;
+    // Delegated constracts cannot use string values, so we use bytes32 values here
+    bytes32 constant public name = "Reputation";
+    bytes32 constant public symbol = "REP";
```

Comment:
Also, `constant public` is redundant I believe (though, if its not I need to fixup a bunch of the stuff I wrote).